x/feemarket/keeper/eip1559.go

```diff
@@ -58,13 +58,15 @@ func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 	parentGasTarget := parentGasTargetBig.Uint64()
 	baseFeeChangeDenominator := new(big.Int).SetUint64(uint64(params.BaseFeeChangeDenominator))
 
-	// If the parent gasUsed is the same as the target, the baseFee remains unchanged.
+	// If the parent gasUsed is the same as the target, the baseFee remains
+	// unchanged.
 	if parentGasUsed == parentGasTarget {
 		return new(big.Int).Set(parentBaseFee)
 	}
 
 	if parentGasUsed > parentGasTarget {
-		// If the parent block used more gas than its target, the baseFee should increase.
+		// If the parent block used more gas than its target, the baseFee should
+		// increase.
 		gasUsedDelta := new(big.Int).SetUint64(parentGasUsed - parentGasTarget)
 		x := new(big.Int).Mul(parentBaseFee, gasUsedDelta)
 		y := x.Div(x, parentGasTargetBig)
```

Comment:
There could be a scenario where the MinGasPrice param had changed even though the parentGasUsed is at the current gasTarget. Saying this because we are not setting the lower bound with the MinGasPrice here. Having said that, I do not think its a big issue, This might be the correct approach.

Reply:
@fedekunze I was thinking the same! But then I thought about how the min gas price is about reducing overall spam txs and a param change will rarely occur. So I don't think it's an issue